openapi.yaml

```diff
@@ -1128,6 +1128,13 @@ components:
           default: false
           examples: 
             - false
+        category:
+          description: A category of endgrate types. Adds any schemas from the category that are not manually defined
```

Comment:
Description can be "An integration category. If specified, all default schemas from the category will be added for this session."